doc/install/upgrading-ceph.rst

```diff
@@ -742,8 +742,8 @@ Under those directories, the keyring should be in a file named ``keyring``.
 
 .. _Monitor Config Reference: ../../rados/configuration/mon-config-ref
 .. _Joao's blog post: http://ceph.com/dev-notes/cephs-new-monitor-changes 
-.. _Ceph Authentication: ../../rados/operations/authentication/
-.. _Ceph Authentication - Backward Compatibility: ../../rados/operations/authentication/#backward-compatibility
+.. _Ceph Authentication: ../../rados/operations/user-management/
```

Comment:
might want to update the place referencing this section as well to reflect the updated name of the "Authentication" section. like
```diff
diff --git a/doc/install/upgrading-ceph.rst b/doc/install/upgrading-ceph.rst
index 3ff63e2..b7f8889 100644
--- a/doc/install/upgrading-ceph.rst
+++ b/doc/install/upgrading-ceph.rst
@@ -174,8 +174,8 @@ daemon cannot participate in the new protocol).  We recommend not waiting for an
 extended period of time between ``ceph-mon`` upgrades. See `Upgrading 
 Monitors`_ for details.
 
-.. note:: See the `Authentication`_ section and the 
-   `Ceph Authentication - Backward Compatibility`_ for additional information
+.. note:: See the `User Management`_ section and the 
+   `User Management - Backward Compatibility`_ for additional information
    on authentication backward compatibility settings for Bobtail.
 
 Once you complete the upgrade of your monitors from Argonaut to
```